GetFed/GetFed/Controllers/FoodSearchViewController.swift

```diff
@@ -57,6 +59,37 @@ class FoodSearchViewController: UIViewController {
         }
         self.foodTableView.reloadData()
     }
+    
+    func deleteFoodEntry(deleteAlertAction: UIAlertAction) {
+        if let indexPath = foodEntryIndexPath {
+            foodTableView.beginUpdates()
+            let foodEntryToDelete = foodArray[indexPath.row]
+            CoreDataManager.sharedManager.deleteEntryByLabel(foodLabel: foodEntryToDelete.label)
+            
+            foodArray.remove(at: indexPath.row)
+            foodTableView.deleteRows(at: [indexPath], with: .fade)
+            
+            CoreDataManager.sharedManager.saveContext()
+            foodTableView.endUpdates()
+        }
+    }
+    
+    func cancelFoodEntryDeletion(cancelAlertAction: UIAlertAction) {
+        foodEntryIndexPath = nil
+    }
+    
+    func deleteConfirmationAlert(for entry: Food) {
+        let alert = UIAlertController(title: "Delete Food Entry?", message: "Are you sure that you want to delete this food entry for \(entry.label)?", preferredStyle: .alert)
```

Comment:
You could check the UI design guidelines from Apple about whether you'd want to use an `.alert` or `.actionSheet` style here.

Reply:
I've pushed up changes. I changed the preferred style to an action sheet. I also cleaned up the message. (Apple used an action sheet as an example here: https://developer.apple.com/documentation/uikit/windows_and_screens/getting_the_user_s_attention_with_alerts_and_action_sheets)